init.sql

```diff
@@ -12,21 +17,18 @@ CREATE SCHEMA api AUTHORIZATION lemmingpants;
 CREATE TYPE model.state AS ENUM ('init', 'active', 'done');
 
 -- The data model for Lemmingpants
-\i sql/model.sql
-\i sql/model/agenda_item.sql
-\i sql/model/speaker_queue.sql
-\i sql/model/attendee.sql
-\i sql/model/speaker.sql
+\i /sql/model.sql
```

Comment:
Absolute paths vill wreak havoc if you don't run this file from a Docker container. Please make it work with relative paths instead so I can keep using my setup that does'nt use Docker. :)